ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/432__data-science-pipelines-tekton.robot

```diff
@@ -27,20 +27,27 @@ Verify Ods Users Can Create And Run A Data Science Pipeline Using The Kfp_tekton
     ...         Tier1
     ...         ODS-2203
 
-    End To End Pipeline Workflow Using Kfp_tekton
+    End To End Pipeline Workflow Using Kfp Tekton
     ...    username=${TEST_USER.USERNAME}
     ...    password=${TEST_USER.PASSWORD}
     ...    project=${PROJECT_NAME}
     ...    python_file=flip_coin.py
     ...    method_name=flipcoin_pipeline
     ...    status_check_timeout=440
-    End To End Pipeline Workflow Using Kfp_tekton
+    End To End Pipeline Workflow Using Kfp Tekton
     ...    username=${TEST_USER.USERNAME}
     ...    password=${TEST_USER.PASSWORD}
     ...    project=${PROJECT_NAME}
     ...    python_file=upload_download.py
     ...    method_name=wire_up_pipeline
     ...    status_check_timeout=440
+    End To End Pipeline Workflow Using Kfp Tekton
```

Comment:
I think this test should be in this folder ods_ci/tests/Tests/650__distributed_workloads
You have to add
```
    Skip If Component Is Not Enabled    ray
    Skip If Component Is Not Enabled    codeflare
```

Reply:
We are testing "Pipelines" here. It is function inside Pipelines.
"650__distributed_workloads" -> Should be related only with distributed workloads

Reply:
That's fine. In this case, can you create a new test in the same test suite and skip it if the components are not enabled?